src/main/java/com/google/sps/servlets/PotentialMatchesServlet.java

```diff
@@ -91,7 +142,6 @@ private String loadUserPotentialMatch(String userID) {
   * @return the set of user nodes for all current app users
   */
   private ImmutableSet<UserNode> createUserNodes() {
```

Comment:
nit: since this isn't actually creating anything in the datastore, I would rename to e.g. `getAllUserNodes`

(even though it's "creating" Java objects, verbs used in a servlet should generally describe the long-term effects, of which there are none here)

Reply:
Got it!